### PR TITLE
Patch neuron to fix load balance issue

### PR DESCRIPTION
**Pull request by iomaganaris**
Applies patch to version 7.8.0a of neuron which fixes the load balancing calculation issue

### CODE REVIEW

File: var/spack/repos/builtin/packages/neuron/package.py [hunk 1]
```diff
@@ -26,8 +26,11 @@ class Neuron(Package):
     # Patch which reverts 81a7a39 for numerical compat
     patch('revert_Import3d_numerical_format.patch', when='@7.8.0:')
 
+    # Patch which applies load balancing fix
+    patch('apply_79a4d2af_load_balance_fix.patch', when='@7.8.0:')
+
     version('develop', branch='master')
-    version('7.8.0a',  commit='92a208b', preferred=True)
+    version('7.8.0b',  commit='92a208b', preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neuron/package.py:33]: `preferred` seems to be redundant, since the newest version is always preferred.

**Review by matz-e — APPROVED** [review]
LGTM

**Review by ferdonline — CHANGES_REQUESTED** [review]
it might be preferable to release the spack recipe fixes for neurodamus at the same time

**Comment by iomaganaris** [comment]
> it might be preferable to release the spack recipe fixes for neurodamus at the same time

I will abort this PR and create a new one for the Simulation Stack January Release to add everything needed in the release

**Pull request closed without merging**